ui/src/components/integrationFormFields/integrationFormFields.tsx

```diff
@@ -33,7 +45,7 @@ export const IntegrationFormFields: FC<IntegrationFormFieldsInterface> = (props)
         isRequired
       >
         <FieldErrorHint provideHint={false}>
-          <FieldText maxLength={55} defaultWidth={false} />
+          <FieldText maxLength={MAX_LENGTH} defaultWidth={false} />
```

Comment:
Do we need to remove `maxLength` property for the fields as discussed previously?
In case it is set, user input will be cut while doing copy-paste.
Discuss with BA.